docs/resources/conformance-statement.md

```diff
@@ -69,7 +69,7 @@ The following DICOM elements are required to be present in every DICOM file atte
 - SOPClassUID
 - PatientID
 
-> Note: All identifiers must be between 1 and 64 characters long, and only contain alpha numeric characters or the following special characters: `.`, `-`.
+> Note: All UIDs must be between 1 and 64 characters long, and only contain alpha numeric characters or the following special characters: `.`, `-`. PatientID is validated based on its LO VR type.
```

Comment:
since we're updating this language, we should check this - I think UIDs can only contain numeric and `.`. Just ran some tests where UIDs with alpha or `-` failed.